modules/react/src/deckgl.ts

```diff
@@ -187,8 +223,7 @@ const DeckGL = forwardRef((props, ref) => {
 
   useImperativeHandle(ref, () => getRefHandles(thisRef), []);
 
-  const {viewManager} = thisRef.deck || {};
-  const currentViewports = viewManager && viewManager.getViewports();
```

Comment:
@Pessimistress This change has introduced a regression which causes React apps to crash on initialization. It happens because there is no guarantee that `viewManager` is initialized when `thisRef.deck` is. The simplest fix would be to add back the old check, but this would require exposing `viewManager` as public.

Reply:
Fix without exposing `viewManager`: https://github.com/visgl/deck.gl/pull/7049